docs/topics/db/optimization.txt

```diff
@@ -396,9 +396,14 @@ Note that there are a number of :meth:`caveats to this method
 <django.db.models.query.QuerySet.bulk_update>`, so make sure it's appropriate
 for your use case.
 
+.. _insert-many-to-many-in-bulk:
+
 Insert in bulk
 --------------
 
+One source object, many target objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I don't think that we need extra subsections.

Reply:
Removed.